src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents an person's contact.
+ * Guarantees: immutable;
+ */
+public class Contact {
+    private final String value;
+    protected boolean isPrivate;
+
+    Contact(String value, boolean isPrivate){
+        this.value = value.trim();
```

Comment:
Here it would be good if you can do validation for the value as well. (You may refer to week 6's LOs on OOP)